docs/introduction.md

```diff
@@ -0,0 +1,155 @@
+# Introducing Ansible Playbook Bundles and Ansible Service Broker for OpenShift
+
+Ansible Playbook Bundle (APB) is a new method for defining and distributing container applications in OpenShift.
+It will leverage Ansible to create a standard path for transitioning from easy to complex deployments. Imagine
+you install a prepackaged application on your cluster and then learn you need to customize the deployment to
+make it production ready. What if you could peel back the cover in a sense and tweak the packaged files to
+satisfy your needs?  Ansible Playbook Bundle is designed to make this transition from simple to complex workflows
+possible.
+
+To introduce the Ansible Playbook Bundle concept we will cover:
+
+  1.) Service Catalog and Ansible Service Broker workflow
+
+  2.) Ansible Playbook Bundle overview
+
+  3.) Ansible Playbook Bundle workflow
+
+    a.) Preparing an Ansible Playbook Bundle
+
+    b.) Packaging an Ansible Playbook Bundle
+
+    c.) Deploying an Ansible Playbook Bundle
+
+### Ansible Service Broker
+
+In December 2016 CloudFoundry [announced](https://www.cloudfoundry.org/open-service-broker-api-launches-as-industry-standard/) the open sourcing of its service broker APIs, [Open Service Broker API](https://www.openservicebrokerapi.org/).
+The Open Service Broker API defines methods for creating a broker, an entity responsible for delivering
+applications or services to a cloud platform. The Ansible Service Broker is a specialized broker created
+for OpenShift to manage Ansible Playbook Bundles.
+
+Users will interact with the [Service Catalog](https://github.com/kubernetes-incubator/service-catalog) to obtain a list of available applications and invoke operations
+for provisioning, deprovisioning, binding, and unbinding to an application. The Service Catalog will rely on
+a collection of brokers to handle details associated with its applications. The diagram below illustrates a
+high level concept of the workflow.
+
+![Overview](ansible-service-broker-overview.png)
+
+**Service Catalog to Service Broker Workflow**
+
+  1.) User requests list of available applications from the Service Catalog
+
+  2.) Service Catalog asks the Ansible Service Broker for available applications
+
+  3.) Ansible Service Broker talks to a container registry to learn which APBs are available
+
+  4.) User issues a command to provision a specific APB
+
+  5.) Provision request makes its way to the Ansible Service Broker which fulfills the command by invoking the
+      provision method on the APB
```

Comment:
I would reformat this to be single spaced, then just use the 1,2,3--- button. i.e. 

```
1. User requests list of available applications from the Service Catalog
2. Service Catalog asks the Ansible Service Broker for available applicatins
```